lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -328,6 +328,7 @@
       <display file="ucsc/maf_customtrack.xml"/>
     </datatype>
     <datatype extension="mtx" type="galaxy.datatypes.tabular:MatrixMarket" display_in_upload="true"/>
+    <datatype extension="cmap" type="galaxy.datatypes.tabular:CMAP" display_in_upload="true"/>
```

Comment:
@jaidevjoshi83 I think you need to register the sniffer in this file below as well.

Reply:
Can you maybe add a description and a `description_url` to https://bionanogenomics.com/wp-content/uploads/2017/03/30039-CMAP-File-Format-Specification-Sheet.pdf